### ISSUE TITLE
D.7.7 Item Base - Film or Video

### ISSUE THREAD

**Issue by paulduchesne** [FIAF/fiaf-cataloguing-manual#82]
@ladislav-nfa @natashafairbairn @heftberger 

I have been doing some new work on converting the FIAFcore vocabularies into SKOS, starting with D.7.7, and was wondering if we could look at this critically, maybe in relation to other resources?

The current list:

Acetate
Acrylic
CTA
Diacetate
Mainly safety
Mainly nitrate
Mixed
Mylar
Nitrate
Polyester
PVC
Safety
Video
Vinyl

`Acetate`, `Nitrate`, `Polyester` seem consistent and agreed upon terms, including from the [Terms and Methods for Technical Archiving](https://www.fiafnet.org/images/tinyUpload/2021/07/Terms-Methods-for-Technical-Archiving-RED.pdf).

`Diacetate` and `Triacetate` (here as "CTA") seem like common additions for further granularity, although raise the question if `Diacetate` is a "type" of `Acetate`, or they are siblings.

`Safety` seems like a colloquial term for `Acetate` or Acetate/Poly, do not believe it is distinct enough to justify a distinct term.

`Mixed`, `Mainly safety`, `Mainly nitrate` seem to exist for instance in which the values are used in a single-value dropdown, for multi-value selects it is easy enough to choose the relevant combination types.

`Video`, I would argue, is not a base.

My understanding was that `Polyester` is the colloquial for `Mylar`, or at the very least a broader term, but not an expert here at all.

Are there examples of `Acrylic` or `Vinyl` Film or Video base, as this is how D.7.7 is explicitly defined?

**Comment by heftberger** [comment]
I have adjusted accordingly. Please see if that is ok. We could also discuss moving the examples into the main text rather than having an appendix with four entries.

**Comment by paulduchesne** [comment]
Thank you for actioning these @heftberger. And yes, good question if such example with minimal vocabularies would be better served with an "e.g." in the main text.

**Comment by heftberger** [comment]
We discussed today, that it would be good to have this list as a value list because it would also/does also appear in the FIAF Core ontology.

**Comment by natashafairbairn** [comment]
We've got 234,365 Items in our database that have base of "Safety" - don't know if other archives have used it extensively too. It may have been used where the exact type of acetate was not identified (and where it could be one of several types) or following what was on the film stock itself. Interesting article here https://www.nedcc.org/free-resources/preservation-leaflets/5.-photographs/5.1-a-short-guide-to-film-base-photographic-materials-identification,-care,-and-duplication#:~:text=Acetate%20Film%20Base,known%20as%20%22Safety%22%20film. 

We also have 928,988 Items with base Video, so would have to double check with technicians what they mean by that (would Videotape be a base?)

We've only got 5 Items using Acrylic and 5 using Vinyl (older legacy records) so could potentially delete those from list of examples in Manual.

Current BFI base list terms are: Acetate, Acrylic, CTA, Mainly Safety, Mixed, Mylar, Nitrate, Polyester, PVC, Safety, Video, Vinyl

I think the original examples list in the Manual was taken mainly from the BFI one (which in turn would have been created from legacy historical terms used in that field from old database and typed/handwritten entries).

**Comment by ladislav-nfa** [comment]
FIAF Treasures vocabulary introduces these terms:

Acetate
Diacetate
Magnetic tape
Nitrate
Ozaphan
Polyester
Triacetate

**Comment by ladislav-nfa** [comment]
(Perhaps we could consider using FIAF Treasures vocabularies as a potential reference framework?)

**Comment by ladislav-nfa** [comment]
(I realize my comment might be coming late - but working currently on our FIAR Treasures export made me notice their vocabularies, which I thought might be worth considering as a reference source.)

**Comment by paulduchesne** [comment]
I think this is a very good point, there should really not be three distinct vocabularies between the Cataloguing Manual, Treasures, FIAFcore, especially as they are all covering the same domain.

In the back of my mind has been the idea of running a survey across institutions to see which terms they are using, and crucially how often those terms are used - has something similar been attempted in the past?

**Comment by heftberger** [comment]
Thank you for this (these) valuable comments!

I have integrated the Treasures list into the one I updated already. I kept the sub-groups (Di- and Triacetate under Acetate), changed Mylar into Ozaphan (because I also think that Ozaphan is the more wide spread term) and have added Magnetic Tape. I would not add "Safety". I know that databases are probably full of a term like that, but I would personally try to get rid of it, because it is used for anything not Nitrate :-). useful for making decisions on storage. But too unspecific and I would personally not recommend it. Mapping might be another issue (would we map it to Acetate or Polyester?)
I vote for taking the list as it is now.

**Comment by natashafairbairn** [comment]
I'm re-opening this issue because the value list got lost somewhere in making previous changes and deleting from former Appendix D 7.7. There is currently no list of examples under Base in the Elements of an Item section of the Items chapter.

Based on previous conversations in this issue previously and in other related issues I suggest the following example list:

Acetate
Diacetate
Triacetate (aka CTA)
Nitrate
Polyester
Magnetic tape

In answer to a couple of previous questions above Mylar and Ozaphan are brands of polyester film:
Mylar®: A registered trademark, originally by DuPont, for a high-quality, durable polyester film.
Ozaphan®/Hostaphan®: A brand name primarily used for polyester (PET) film produced by Hoechst (now often associated with Mitsubishi Polyester Film), which serves the same functional purposes as Mylar, including electrical insulation and imaging.

There's nothing wrong with still including them in list if desired, but this is meant to be just a sample list not a definitive full one. I haven't added anything into Develop yet.

**Comment by heftberger** [comment]
I just checked and I don't think the list was lost, it was just boiled down to examples: (e.g. acetate, nitrate, polyester) or a magnet track (e.g. CTA). We can add Mylar and Ozaphan, but I actually think it is ok like this.

Do you agree? If not, we can add the full list in the brackets.

**Comment by heftberger** [comment]
@natashafairbairn

**Comment by heftberger** [comment]
This seems merged.

**Comment by paulduchesne** [comment]
The section being discussed? https://fiaf-cataloguing-manual.s3-website.fr-par.scw.cloud/items/elements_of_a_moving_image_item/#base

**Comment by heftberger** [comment]
Yes, and I think we don't have to do anything more to it. We integrated examples from the list and I think this part is fine now. So I was wondering if we still need this issue.
@natashafairbairn @natashafairbairn

**Comment by heftberger** [comment]
Discussed and we can close it.
